### DISCUSSION TITLE
Different themes for different users

### DISCUSSION THREAD

**Discussion by ghubnab99** [vercel/community#3233] [Help] [answered]
### Question

Hi,

I believe this maybe a more Next.js/React.js based question, but nevertheless.

We're building something similar to wordpress (but not exactly, it's for a different use case), where the user will have an admin panel as well as a front facing website. They can change the theme of the website from the admin panel. In the admin panel they have a no. of themes to choose from. When they select a specific theme, the UI would be different. 

How can we render different UI for different users based on the theme selection. How can we achieve this?

Thanks

**Comment by amyegan** [comment, marked as answer]
Hi @ghubnab99! You'll need a way to keep track of the logged in user and their preferences. There are a lot of ways to do that, so the right choice for you will depend on the app framework and what tools you're comfortable using.

Since this is more of a code question, I think it would be better to ask this in [Next.js Discussions](https://github.com/vercel/next.js/discussions). The folks over there have a lot of experience working with Next.js specifically. I'm sure they'll have some ideas about how to accomplish this goal. 🙂

**Reply by ghubnab99** [reply]
Thanks a lot @amyegan 🙂
